tool/gen_mackerel_plugin.pl

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env perl
```

Comment:
Do you have any idea when to run this script in our development flow?  Apparently we must run it when packaging/config.json changed, but one might forget to run.  IMO we need some automated process in somewhere (ci, release process, or something).